app/backend/approaches/retrievethenread.py

```diff
@@ -108,35 +92,23 @@ async def run(
         )
 
         # Process results
-        sources_content = self.get_sources_content(results, use_semantic_captions, use_image_citation=False)
-
-        # Append user message
-        content = "\n".join(sources_content)
-        user_content = q + "\n" + f"Sources:\n {content}"
-
-        response_token_limit = 1024
-        updated_messages = build_messages(
```

Comment:
I realized that we do not need to call build_messages in the ask approaches, since it only truncates past messages, and we don't pass in chat history to these. So we remove the call and assume that the /ask questions always fit in the context window.